jupyterhub/app.py

```diff
@@ -1457,14 +1457,12 @@ def init_internal_ssl(self):
             self.internal_ssl_cert = internal_key_pair['files']['cert']
             self.internal_ssl_ca = self.internal_trust_bundles[hub_name]
 
-            # Configure the AsyncHTTPClient. This will affect anything using
-            # AsyncHTTPClient.
-            ssl_context = make_ssl_context(
-                self.internal_ssl_key,
-                self.internal_ssl_cert,
-                cafile=self.internal_ssl_ca,
-            )
-            AsyncHTTPClient.configure(None, defaults={"ssl_options": ssl_context})
+        # Configure the AsyncHTTPClient. This will affect anything using
+        # AsyncHTTPClient.
+        self.ssl_context = make_ssl_context(
+            self.internal_ssl_key, self.internal_ssl_cert, cafile=self.internal_ssl_ca
+        )
+        AsyncHTTPClient.configure(None, defaults={"ssl_options": self.ssl_context})
```

Comment:
This is no longer part of the `if` clause as it were before, is that as it should be?